solr/core/src/test/org/apache/solr/handler/admin/IndexSizeEstimatorTest.java

```diff
@@ -227,8 +227,9 @@ public void testIntegration() throws Exception {
           (k, v) -> {
             double size = fromHumanReadableUnits((String) v);
             double sampledSize = fromHumanReadableUnits((String) sampledFieldsBySize.get(k));
-            assertNotNull(
```

Comment:
Why did this test change?

Reply:
According to error-prone:

> error: [ImpossibleNullComparison] This value cannot be null, and comparing it to null may be misleading.

This is because primitive data types have always a default value. Resolving this to focus on the root discussion.